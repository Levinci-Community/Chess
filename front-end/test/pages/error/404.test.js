import React from "react";
import { render } from "@testing-library/react";
import NotFoundPage from "../../../src/pages/error/404";
import { BrowserRouter as Router } from "react-router-dom";

jest.mock("react-i18next", () => ({
    useTranslation: () => ({
      t: (key) => key,
    }),
  }));
  
  test("renders NotFoundPage snapshot", () => {
    const { container } = render(
      <Router>
        <NotFoundPage />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });