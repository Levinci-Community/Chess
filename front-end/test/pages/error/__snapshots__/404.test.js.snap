// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NotFoundPage snapshot 1`] = `
<div>
  <div
    class="chakra-container css-vl7sen"
  >
    <div
      class="chakra-stack css-1kbzohu"
    >
      <h2
        class="chakra-heading css-zov4ca"
      >
        404 Not Found
      </h2>
      <p
        class="chakra-text css-0"
      >
        The page you are looking for might be in another castle.
         
        <a
          class="chakra-link css-1o61trh"
        >
          Go back home
        </a>
      </p>
    </div>
  </div>
</div>
`;
